assets/terraform-modules/worker/variables.tf

```diff
@@ -0,0 +1,69 @@
+# Required variables.
+variable "ssh_keys" {
+  type        = list(string)
+  description = "List of SSH public keys for user `core`. Each element must be specified in a valid OpenSSH public key format, as defined in RFC 4253 Section 6.6, e.g. 'ssh-rsa AAAAB3N...'."
+  default     = []
+}
+
+# Optional variables.
+variable "count_index" {
+  type        = number
+  description = "Number of nodes to create."
+  default     = 1
+}
+
+variable "cluster_dns_service_ip" {
+  type        = string
+  description = "IP address of cluster DNS Service. Passed to kubelet as --cluster_dns parameter."
+  default     = "10.3.0.10"
+}
+
+variable "clc_snippets" {
+  type        = list(string)
+  description = "Extra CLC snippets to include in the configuration."
+  default     = []
+}
+
+variable "kubelet_image_name" {
+  type        = string
+  description = "Source of kubelet Docker image"
```

Comment:
Same period at the end of the description here and other occurrences in the same file.